conda_build/conda_interface.py

```diff
@@ -22,7 +22,7 @@
 from conda.signature import KEYS, KEYS_DIR, hash_file, verify  # NOQA
 from conda.utils import human_bytes, hashsum_file, md5_file, memoized, unix_path_to_win, win_path_to_unix, url_path  # NOQA
 import conda.config as cc  # NOQA
-from conda.config import sys_rc_path  # NOQA
+from conda.config import rc_path  # NOQA
```

Comment:
Is this a change between conda 4.1 and 4.2?  Are you sure that this import is the same on both?  If not, please do the import differently in each section below.